proto/warden/async/v1beta1/plugin.proto

```
@@ -2,7 +2,10 @@ syntax = "proto3";
 
 package warden.async.v1beta1;
 
+import "amino/amino.proto";
```

Buf (1.47.2)

import "amino/amino.proto": file does not exist

(COMPILE)